core-scala/src/test/scala/com/baeldung/scala/GenericsIntro.scala

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.scala
+
+object GenericsIntro {
+  case class Apple(name: String)
+
+  def main(args: Array[String]): Unit = {
+    NonGenericWay.run()
+    GenericWay.run()
+  }
+
+  object NonGenericWay{
+    case class MagicHat(magic: AnyRef)
+
+    def run() = {
+      val rabbitHat = MagicHat(Rabbit(2))
+      val rabbit: Rabbit = rabbitHat.magic.asInstanceOf[Rabbit]
+      println(rabbit.cuteness)
+    }
+  }
+
+  object GenericWay{
```

Comment:
I notice an inconsistency with the use of `{` after the name. Sometimes there's a space and sometimes there isn't. Which is the standard? is it `Name {` like in Java?